package test2;

import java.io.IOException;

import org.testng.annotations.Test;

import testngFiles.TestBase;

public class fortest2 extends TestBase {
	
	
	@Test
	public void ploan() throws IOException
	{
		//System.out.println("good");
		Login();
		System.out.println("good2");
		System.out.println("good3");
	}
}
